Constant 4294967295 overflows int on 32-bit builds

When compiling for a 32-bit platform I get:

`..\..\go\pkg\mod\github.com\open2b\scriggo@v0.52.2\internal\compiler\builder.go:580:18: constant 4294967295 overflows int`

The problematic code is:

`if len(fn.Body) > math.MaxUint32 {`

len(n) returns an int, not an unsigned int, which raises the compiler error.

--- a/internal/compiler/builder.go
+++ b/internal/compiler/builder.go
@@ -577,8 +577,8 @@ func (fb *functionBuilder) end() {
 	// if len(fn.Body) == 0 || fn.Body[len(fn.Body)-1].Op != runtime.OpReturn {
 	// 	fb.emitReturn()
 	// }
-	if len(fn.Body) > math.MaxUint32 {
-		panic(newLimitExceededError(fn.Pos, fn.File, "instructions count exceeded %d", math.MaxUint32))
+	if int64(len(fn.Body)) > math.MaxUint32 {
+		panic(newLimitExceededError(fn.Pos, fn.File, "instructions count exceeded %d", uint32(math.MaxUint32)))
 	}
 	for addr, label := range fb.gotos {
 		i := fn.Body[addr]

